### PR TITLE
Bump more references of dandiset to dataset

### PR DESCRIPTION
**Pull request by aaronkanzer**

### CODE REVIEW

File: dandiapi/api/templates/api/account/base.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
 
-  <title>DANDI Archive</title>
+  <title>LINC Archive</title>
```
Comment (kabilar) [inline dandiapi/api/templates/api/account/base.html:12]:
```suggestion
  <title>LINC Platform</title>
```

File: web/src/components/DandiFooter.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
       </cookie-law>
       <v-row>
         <v-col offset="2">
-          &copy; 2019 - 2023 The DANDI Team<br>
+          &copy; 2024 The LINC Archive Team<br>
```
Comment (kabilar) [inline web/src/components/DandiFooter.vue:18]:
```suggestion
          &copy; 2023 - 2024 LINC Team<br>
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Thank you @aaronkanzer.  A few minor comments above.

File: web/src/components/DandiFooter.vue (hunk 2 above)
Comment (aaronkanzer) [reply web/src/components/DandiFooter.vue]: Updated!

File: dandiapi/api/templates/api/account/base.html (hunk 1 above)
Comment (aaronkanzer) [reply dandiapi/api/templates/api/account/base.html]: Updated!

**Review by kabilar — APPROVED** [review]

**Pull request merged**